android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -371,18 +371,22 @@ public void onLeScan(final BluetoothDevice device, final int rssi, final byte[]
           @Override
           public void run() {
             UriBeacon uriBeacon = UriBeacon.parseFromBytes(scanResult.getScanRecord().getBytes());
```

Comment:
Yeah but to access the object you put inside it:
```java
scanResult.getScanRecord()
scanResult.getDevice()
scanResult.getRssi()
```
You have direct access to those objects (`scanRecord`, `rssi`, `device`) so there is no need for the `ScanResult`.